lib/pages/filter/view/filter_dropdown.dart

```diff
@@ -62,7 +62,7 @@ class _FilterDropdownState extends State<FilterDropdown> {
                         child: Padding(
                           padding: EdgeInsets.only(
                               top: 8, left: widget.leftPadding ?? 0.0),
-                          child: Text(node.name),
+                          child: Text(node.localizedName(context)),
```

Comment:
The dropdown is not the only place where this should be called. We also have relevance_picker and filter_page which display the filter node names.